pkg/cmd/server/origin/audit.go

```diff
@@ -0,0 +1,24 @@
+package origin
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/golang/glog"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+func basicAuditHandler(handler http.Handler, contextMapper kapi.RequestContextMapper) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		ctx, _ := contextMapper.Get(req)
+		user, _ := kapi.UserFrom(ctx)
+		namespace := kapi.NamespaceValue(ctx)
+		usernamespace := user.GetName()
+		if len(namespace) != 0 {
+			usernamespace = fmt.Sprintf("%s@%s", user.GetName(), namespace)
+		}
+		glog.Infof("%s - %s - %v", req.Method, usernamespace, req.URL)
```

Comment:
> I had that, but then I figured out I'd go with audit.go file name which will do just that  Field names sgtm.

That is dependent on a particular glog formatting choice, which is theoretically changable. I'd pay for the extra characters in this method.